modules/update-upgrading-cli.adoc

```diff
@@ -4,6 +4,25 @@
 // * updating/updating-cluster-rhel-compute.adoc
 
 [id="update-upgrading-cli_{context}"]
+= Optional: Changing the update server by using the CLI
+
+If you have an OpenShift Update Service (OSUS) installed and configured locally, you can use it. To consume your local OSUS, you must change the upstream. The default `upstream` is https://api.openshift.com/api/upgrades_info/v1/graph.
```

Comment:
The "you can use it" part catches my eye because it doesn't provide the most information that is possible.  Perhaps combine with the sentence that follows.

"If you have an OpenShift Update Service (OSUS) installed and configured locally, you must set the URL for the server as the `upstream` to use the local server during updates. The default value for `upstream` is \https://..."

Add a `\` before the https so that it is not interpreted as a URL that readers can click.

Reply:
Done